plugin/completion.py

```diff
@@ -278,10 +331,12 @@ def run(self, edit: sublime.Edit, event: Optional[dict] = None) -> None:
 
 class LspSelectCompletionCommand(LspTextCommand):
 
-    completions = {}  # type: Dict[SessionName, List[CompletionItem]]
+    completions = {}  # type: Dict[SessionName, CompletionsStore]
 
     def run(self, edit: sublime.Edit, index: int, session_name: str) -> None:
-        item = LspSelectCompletionCommand.completions[session_name][index]
+        item = LspSelectCompletionCommand.completions[session_name]['items'][index]
+        item_defaults = LspSelectCompletionCommand.completions[session_name].get('itemDefaults', {})
+        item = completion_with_defaults(item, item_defaults)
```

Comment:
If you change `CompletionsStore` to a tuple then this can be written in a cleaner way:

```suggestion
        items, item_defaults = LspSelectCompletionCommand.completions[session_name]
        item = completion_with_defaults(items[index], item_defaults)
```